config/routes.rb

```diff
@@ -16,6 +16,8 @@
   post '/api/usps_locations' => 'idv/in_person/public/usps_locations#index'
   match '/api/usps_locations' => 'idv/in_person/public/usps_locations#options', via: :options
 
+  post '/api/webhooks/socure/event' => 'socure_webhook#on_event'
```

Comment:
IMO I'd just go with the default action name of `#create`? 🤷

Reply:
OK.

Reply:
Changed.